src/main/java/edu/ucdavis/dss/ipa/api/components/course/CourseViewController.java

```diff
@@ -38,6 +42,7 @@
 @RestController
 @CrossOrigin
 public class CourseViewController {
+	private static final Logger log = LoggerFactory.getLogger("edu.ucdavis.dss.dw.DwClient");
```

Comment:
1. Do not use the `edu.ucdavis.dss.dw.DwClient` logger. That is for bugs in the package `edu.ucdavis.dss.dw.DwClient`. Change this line to omit specifying a logger: `private static final Logger log = LoggerFactory.getLogger();`

2. Add whitespace after this line. Initializing the logger and setting up the injectors are two separate "paragraphs" of code and therefore need whitespace.

Reply:
If I omit the specifying logger, I get the following error "error: no suitable method found for getLogger(no arguments)". Is there another parameter I can use for logger or should I not use the logger here to display the error at line: 565?